.github/workflows/ci.yml

```diff
@@ -67,3 +67,12 @@ jobs:
       - name: Deploy prod API docs
         if: ${{ github.ref == 'refs/heads/main' && matrix.docsTarget }}
         run: npx vercel deploy build/apidocs -t ${{ secrets.VERCEL_TOKEN }} --name python --scope temporal --prod --yes
+
+  # Runs the sdk features repo tests with this repo's current SDK code
+  sdk-features-tests:
+    uses: temporalio/sdk-features/.github/workflows/python.yaml@main
```

Comment:
So I assume that, if I have breaking changes that would break SDK features, I would point to a branch on there until I release the SDK?

Reply:
That's definitely the most obvious solution for now. If this turns out to be a repeated pain, perhaps we design something easier.